integration_tests/event/test_delete.py

```diff
@@ -0,0 +1,89 @@
+from fastapi.testclient import TestClient
+from fastapi import status
+from sqlalchemy.orm import Session
+from app.clients.db.models.law_policy import FamilyEvent, EventStatus
+from integration_tests.setup_db import setup_db
+import app.repository.event as event_repo
+
+
+def test_delete_event(client: TestClient, test_db: Session, admin_user_header_token):
+    setup_db(test_db)
+    response = client.delete("/api/v1/events/E.0.0.2", headers=admin_user_header_token)
+    assert response.status_code == status.HTTP_200_OK
+    assert test_db.query(FamilyEvent).count() == 3
+    assert (
+        test_db.query(FamilyEvent)
+        .filter(FamilyEvent.status == EventStatus.DELETED)
+        .count()
+        == 1
+    )
+    assert test_db.query(FamilyEvent).count() == 3
```

Comment:
Maybe we should also count the number with the status of DELETE set in all these tests too?